frontend/src/utils/constants.ts

```diff
@@ -19,7 +19,7 @@ export const ERC20_ABI = [
   'event Transfer(address indexed from, address indexed to, uint256 value)',
 ];
 
-export const ERC721_ABI = ['function name() view returns (string)', 'function symbol() view returns (string)'];
+export const TOKEN_ABI = ['function name() view returns (string)', 'function symbol() view returns (string)'];
```

Comment:
We already have `ERC20_ABI` so we don't need to add `TOKEN_ABI`

Reply:
True, I had that originally, but thought it might be clearer to separate out into a separate ABI even though it is captured by the ERC20. I will remove and change to the ERC-20 abi